Airports/EGSS/Ground_Network.txt

```diff
@@ -1,10 +1,153 @@
+COORD:N051.52.59.922:E000.13.56.794
+COORD:N051.52.59.691:E000.13.57.106
+COORD:N051.52.59.379:E000.13.58.260
+COORD:N051.52.58.465:E000.14.01.763
+EXIT:22:LR:LEFT:45
+COORD:N051.52.57.399:E000.13.52.418
+COORD:N051.52.56.667:E000.13.51.391
+COORD:N051.52.56.080:E000.13.50.610
+COORD:N051.52.55.312:E000.13.49.776
+COORD:N051.52.54.505:E000.13.48.975
+COORD:N051.52.53.456:E000.13.48.093
+COORD:N051.52.52.045:E000.13.47.103
+COORD:N051.52.50.831:E000.13.46.405
+COORD:N051.52.49.517:E000.13.45.745
+COORD:N051.52.48.505:E000.13.45.163
+COORD:N051.52.44.069:E000.13.42.904
+EXIT:22:L:LEFT:25
+COORD:N051.52.47.189:E000.13.37.046
+COORD:N051.52.46.984:E000.13.36.762
+COORD:N051.52.46.841:E000.13.36.627
+COORD:N051.52.46.543:E000.13.36.460
+COORD:N051.52.46.324:E000.13.36.390
+COORD:N051.52.46.075:E000.13.36.414
+COORD:N051.52.45.884:E000.13.36.466
+COORD:N051.52.45.575:E000.13.36.657
+COORD:N051.52.45.312:E000.13.36.937
+COORD:N051.52.43.829:E000.13.39.517
+EXIT:22:K:LEFT:35
+COORD:N051.52.37.521:E000.13.22.577
+COORD:N051.52.37.152:E000.13.22.128
+COORD:N051.52.36.931:E000.13.21.957
+COORD:N051.52.36.658:E000.13.21.727
+COORD:N051.52.36.379:E000.13.21.569
+COORD:N051.52.36.121:E000.13.21.481
+COORD:N051.52.35.825:E000.13.21.396
+COORD:N051.52.35.550:E000.13.21.380
+COORD:N051.52.35.223:E000.13.21.368
+COORD:N051.52.34.943:E000.13.21.424
+COORD:N051.52.34.663:E000.13.21.501
+COORD:N051.52.34.328:E000.13.21.673
+COORD:N051.52.34.025:E000.13.21.873
+COORD:N051.52.33.699:E000.13.22.177
+COORD:N051.52.33.394:E000.13.22.531
+COORD:N051.52.33.243:E000.13.22.754
+EXIT:22:H:LEFT:25
+COORD:N051.52.31.475:E000.13.13.521
+COORD:N051.52.31.219:E000.13.13.199
+COORD:N051.52.30.997:E000.13.13.007
+COORD:N051.52.30.836:E000.13.12.944
+COORD:N051.52.30.585:E000.13.12.887
+COORD:N051.52.30.451:E000.13.12.904
+COORD:N051.52.30.241:E000.13.12.947
+COORD:N051.52.29.994:E000.13.13.073
+COORD:N051.52.29.864:E000.13.13.254
+COORD:N051.52.29.488:E000.13.13.897
+COORD:N051.52.28.136:E000.13.16.153
+EXIT:22:G:RIGHT:25
+COORD:N051.52.31.780:E000.13.13.855
+COORD:N051.52.31.661:E000.13.13.601
+COORD:N051.52.31.526:E000.13.13.273
+COORD:N051.52.31.490:E000.13.13.014
+COORD:N051.52.31.433:E000.13.12.503
+COORD:N051.52.31.447:E000.13.12.224
+COORD:N051.52.31.475:E000.13.11.888
+COORD:N051.52.31.545:E000.13.11.566
+COORD:N051.52.31.685:E000.13.11.174
+COORD:N051.52.31.827:E000.13.10.897
+COORD:N051.52.31.993:E000.13.10.732
+COORD:N051.52.32.410:E000.13.10.374
+COORD:N051.52.32.673:E000.13.10.245
+COORD:N051.52.33.615:E000.13.09.791
+EXIT:04:L:RIGHT:25
+COORD:N051.52.45.170:E000.13.33.964
+COORD:N051.52.45.344:E000.13.34.281
+COORD:N051.52.45.453:E000.13.34.622
+COORD:N051.52.45.531:E000.13.34.955
+COORD:N051.52.45.569:E000.13.35.228
+COORD:N051.52.45.599:E000.13.35.410
+COORD:N051.52.45.602:E000.13.35.573
+COORD:N051.52.45.571:E000.13.35.902
+COORD:N051.52.45.496:E000.13.36.263
+COORD:N051.52.45.337:E000.13.36.894
+COORD:N051.52.43.829:E000.13.39.517
```

Comment:
Similarly - L on 04 is before the Aim Point, so idk if this would break something?

Reply:
Tested - No impact on arrivals.